docs/exclusion.md

```diff
@@ -2,44 +2,65 @@
 
 When performing validations in real world-scenarios, you may need to only run tests of a single field in your suite, or skip certain tests according to some logic. That's why Vest includes `vest.skip()` and `vest.only()`.
 
-`vest.skip()` and `vest.only()` are functions that take a name of the test, or a list of names to either include or exclude fields from being validated. They should be called from the body of `validate` callback, and in order for them to take effect, they should be called before anything else.
+`vest.skip()` and `vest.only()` are functions that take a name of the test, or a list of names to either include or exclude fields from being validated. They should be called from the body of suite callback, and in order for them to take effect, they should be called before anything else.
 
-Fields that were skipped (or not included when `only` was used) will appear in the [result object](./result) in the `skipped` array.
+## Important to know before using exclusion hooks:
+
+When using `vest.only()` or `vest.skip()` you must place them before any of the tests defined in the suite. Hooks run in order of appearence, which means that if you place your `skip` hook after the filed you're skipping - it won't have any effect.
 
 ### Only running specific tests
+
 When validating upon user interactions, you will usually want to only validate the input the user currently interacts with to prevent errors appearing in unrelated places. For this, you can use `vest.only()` with the name of the test currently being validated.
 
 In the example below, we're assuming the argument `fieldName` is being populated with the name of the field we want to test. If none is passed, the call to `only` will be ignored, and all tests will run as usual. This allows us to test each field at a time during the interaction, but test all on form submission.
 
 ```js
-import vest, { validate, enforce, test } from 'vest';
+import vest, { enforce, test } from 'vest';
 
-const v = (data, fieldName) => validate('New User', () => {
+const validate = (data, fieldName) => {
```

Comment:
It's a little bit harder for eyes to see const validate twice, would consider changing them to different things in order to make it easier to read.